flink-container/docker/README.md

```diff
@@ -0,0 +1,44 @@
+# Apache Flink cluster deployment on docker using docker-compose
+
+## Installation
+
+Install the most recent stable version of docker
+https://docs.docker.com/installation/
+
+## Build
+
+Images are based on the official Java Alpine (OpenJDK 8) image. If you want to
+build the flink image run:
+
+    sh build.sh --job-jar /path/to/job/jar/job.jar --image-name flink:job
+
+or
+
+    docker build -t flink .
+
+If you want to build the container for a specific version of flink/hadoop/scala
+you can configure it in the respective args:
+
+    docker build --build-arg FLINK_VERSION=1.0.3 --build-arg HADOOP_VERSION=26 --build-arg SCALA_VERSION=2.10 -t "flink:1.0.3-hadoop2.6-scala_2.10" flink
```

Comment:
Is FLINK_VERSION 1.0.3 only for demo purpose? Can we use a more recent version for demoing?